From 92f05979d5a5a74779f617766011b40711fbf228 Mon Sep 17 00:00:00 2001
From: yoinx <joseph.schubert@gmail.com>
Date: Thu, 9 Feb 2017 23:01:46 -0500
Subject: [PATCH] Add Functionality to control High Brightness Mode [2/3]

This is needed to add the tile as it overlays Frameworks/Base.

Depends on device tree and kernel support'

Change-Id: I325569f1507d9603cbaa1da089f90384cd21b4e9
---
 overlay/common/frameworks/base/packages/SystemUI/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlay/common/frameworks/base/packages/SystemUI/res/values/config.xml b/overlay/common/frameworks/base/packages/SystemUI/res/values/config.xml
index c1ef1d3..0db48a4 100644
--- a/overlay/common/frameworks/base/packages/SystemUI/res/values/config.xml
+++ b/overlay/common/frameworks/base/packages/SystemUI/res/values/config.xml
@@ -26,7 +26,7 @@
 
     <!-- Tiles native to System UI. Order should match "quick_settings_tiles_default" -->
     <string name="quick_settings_tiles_stock" translatable="false">
-        wifi,cell,battery,dnd,flashlight,rotation,bt,airplane,location,hotspot,inversion,saver,work,cast,night,screenshot,adb_network,sync,expanded_desktop,reboot,music,ime,caffeine,navigation_bar,androidauto,heads_up,ambient,nfc,weather,sound,hardware_button
+        wifi,cell,battery,dnd,flashlight,rotation,bt,airplane,location,hotspot,inversion,saver,work,cast,night,screenshot,adb_network,sync,expanded_desktop,reboot,music,ime,caffeine,navigation_bar,androidauto,heads_up,ambient,nfc,weather,sound,hardware_button,high_brightness
     </string>
 
     <!-- Timeouts for brightness dialog to disappear -->
-- 
2.13.0

